roles/common/tasks/ssh.yml

```diff
@@ -14,3 +14,16 @@
             group={{ item.group|default("root") }} mode=0600
   with_items: "{{ common.ssh_private_keys }}"
 
+- name: find ssh private  key files
+  find:
+    path: "/etc/ssh"
```

Comment:
would it make more sense to grep for `-----BEGIN \w+ PRIVATE KEY-----` in the files ?   rather than hope that keys have the _key suffix ?